docs/install.rst

```diff
@@ -279,6 +281,69 @@ If you have already activated the correct environment, but you still do not see
 Frequently asked questions
 --------------------------
 
+**I tried the installation instructions, but I cannot import Pylake inside a Jupyter notebook**
+
+If Python claims there is no package with the name `lumicks.pylake` or it fails with a `DLL load failed` error, even after you're followed the above installation instructions, then please check the following:
+
+* You should be launching the Jupyter Notebook from within the same environment where you installed `pylake`.
+
+* If you already had an installation of Anaconda, and you installed `pylake` into a new environment alongside existing environments, it may be that you have a conflict between multiple Jupyter installations.
+
+In this case, try running the following command from an Anaconda Prompt::
+
+    jupyter kernelspec list
+
+This lists all the kernels that have been installed (thereby making them available to Jupyter).
+
+If the output of this command lists any paths containing `AppData\\Roaming\\jupyter\\kernels` or `C:\\ProgramData\\jupyter\\kernels` (Windows); `.local/share/jupyter/kernels` or `/usr/share/jupyter/kernels` or `/usr/local/share/jupyter/kernels` (Mac/Linux); or `/Library/Jupyter/kernels` (Mac); then invoke::
+
+    python -m ipykernel install --user --name=envname
+
+Where `envname` should be replaced with the name of the environment you are using.
+
+Now restart the Jupyter Notebook, and make sure you open your Notebook using the `pylake` kernel that's now available in the list.
+
+*Why does this work?*
```

Comment:
This is a great approach, to also give the technical explanation in the FAQ, but to clearly mark it as such and to keep it separate from the answer.